pkg/codegen/python/gen_test.go

```diff
@@ -59,164 +56,160 @@ func TestRelPathToRelImport(t *testing.T) {
 }
 
 func TestGeneratePackage(t *testing.T) {
+	// To speed up these tests, we will generate one common
```

Comment:
I found it to help a little bit, though it reduces isolation, I think it's worth it. The test suite rebuilds it on every run of the suite, but that's better than one per test.